frontend-react/.gitignore

```
@@ -22,3 +22,8 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+# local env files
+.env.local
+.env.*.local
+.env
```

以下みたいにしてシンプルに書けないですかね　
```suggestion
.env*
!.env.example
```